lesson26/src/js/login.js

```diff
@@ -87,3 +89,15 @@ for (const input of inputSelector){
 }
 
 togglePasswordButton.addEventListener('click', () => togglePassword(passwordInput,togglePasswordButton));
+
+togglePasswordButton.addEventListener('blur', (event) => {
+    if(isRelatedTarget(event,"current-password")) return;
+
+    validation.checkPassword(passwordInput);
+    changeDisabledStatusSubmitButton();
+});
```

Comment:
```suggestion
togglePasswordButton.addEventListener('blur', (event) => {
    if(isRelatedTarget(event,"current-password")) return;

    validation.removeErrorMessage(passwordInput); //エラーをリセットしたい
    validation.checkPassword(passwordInput);
    changeDisabledStatusSubmitButton();
});
```
何度も申し訳ないです🙇‍♀️
①入力してくださいエラーが出ている状態で
②何かしら入力する
③表示・非表示ボタンを押す
④そのあとinput以外を押す
この時、入力してくださいエラーが出続けていて、バリデーションの内容がユーザーに伝わらないので、この点だけ直すと良いと思ったのですがどうでしょうか？

refactorは必要かもしれませんが、上記コードを書くなどしてエラーメッセージをリセットする必要があるかと思います。あるいは、リセットをcheckPassword(passwordInput)関数内で行うか（こちら未検証です🙇‍♀️）

何度も申し訳ありません。。。



https://user-images.githubusercontent.com/67426438/224870851-fc0a0203-5719-4e60-b5cc-9b4733a8014d.mov

Reply:
具体的なコードをありがとうございます！
こちらこそ、何度も申し訳ありません。。
確かに古いエラーメッセージが残ってしまっているので、新しいエラー内容をユーザーに伝える必要がありました。

ご提案いただいたコードと同様のものに修正いたしましたので、
ご確認いただけると助かります。何度もすみません:pray:
https://github.com/nakagawamai/js-lesson/pull/34/commits/a12197e3c26a037be45cb0d36e7ab51144cafc2f